library/kotlin/io/envoyproxy/envoymobile/Engine.kt

```diff
@@ -27,4 +27,11 @@ interface Engine {
    * This is a noop if called before the underlying EnvoyEngine has started.
    */
   fun flushStats()
+
+  /**
+   * Retrieve the value of all active stats. Note that this function may block for some time.
+   * @param timeout_ms the timeout for how long to wait when fetching stats.
+   * @return The list of active stats and their values, or empty string of the operation failed
+   */
+  fun dumpStats(timeout_ms: Int): String
```

Comment:
I think that we should get rid of this argument (per my previous comment) but if we want to keep it, it should probably follow camel case